src/lib/fetchChatter.ts

```diff
@@ -30,9 +30,9 @@ export class FetchChatter {
             'chatter');
     }
 
-    list(): Promise<any> {
```

Comment:
Yeah, I think we should look to keep fetchChatter the way you have it, and put some consideration into how we can provide that wrapper for devs.